test/test_format.py

```diff
@@ -8,523 +8,736 @@
 
 import pytest
 
+#
+#  Aim of the tests:
+#
+# In these tests we check that the current behavior of makeboxes does not change
+# without noticing that it could affect both compatibility with WL and with
+# mathics-django. Also looking at some issues in the curren behavior regarding
+# the WL standard (for instance, how to represent $a^(b/c)$) and the Mathics
+# own implementation (BoxError raising in some simple conditions).
+# These test should be updated as we fix pending issues.
 
-@pytest.mark.parametrize(
-    ("str_expr", "str_expected", "msg"),
-    [
-        ('"4.32313213213`2"', "\\text{4.32`2}", None),
-    ],
-)
-@pytest.mark.xfail
-def test_makeboxes_standardform_tex_failing(
-    str_expr: str, str_expected: str, msg: str, message=""
-):
-    result = session.evaluate(str_expr)
-    format_result = result.format(session.evaluation, "System`StandardForm")
-    # Atoms are not still correctly processed as BoxConstruct
-    #    assert isinstance(format_result,  BoxConstruct)
-    if msg:
-        assert (
-            format_result.boxes_to_text(evaluation=session.evaluation) == str_expected
-        ), msg
-    else:
-        str_format = format_result.boxes_to_text(evaluation=session.evaluation)
-        assert str_format == str_expected
 
+# Set this to False in case mathml tests must be considered xfail. With True, ensures the
```

Comment:
expected fail or expected skip.  Fail means you deliberately want something to fail. Skip means you don't have the means or resources to do something.

Reply:
@rocky,  in this case, *xfail* means that the core is still failing to fulfill the expected behavior (the one in WMA). The good news is that with the last changes here, we pass all the tests. Of course, we can add more.
Still. we could leave some "fragile" tests (those involving complex mathml/svg/xml code)   marked with xfail. 

 In any case, I have to rewrite this statement in the comment.

Reply:
> @rocky, in this case, _xfail_ means that the core is still failing to fulfill the expected behavior (the one in WMA). The good news is that with the last changes here, we pass all the tests. Of course, we can add more. Still. we could leave some "fragile" tests (those involving complex mathml/svg/xml code) marked with xfail.
> 
> In any case, I have to rewrite this statement in the comment.

My understanding is that things that are not implemented yet are skipped with the reason "not implemented yet", not expected to fail. An "expected to fail" kind of test should never all succeed. Divide by 0 is the kind of expected failure.